### DISCUSSION TITLE
Call for Participation (Product, Project, Robotics)

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/ACE_Framework#130] [Announcements]
We have several new members in the Core Team, including an architect and some prospective lead developers! 

Now, we need another couple of people:

1. A Product Owner or Program Manager - This is someone who understands the overarching purpose of ACE and MURPHIE who can participate with the Architect and TL to guide the project. This is a stand-in for myself (Dave) as I don't have the time to serve as full time PO.
2. A Project Manager (Agile, Scrum) - This is someone who can help with organizing sprints and coordinating between the volunteer developers, TL, and architect(s). 
3. More Roboticists/Mechatronics/Fabricators - We have a few folks who are in EE (electrical engineering) with specializations in robotics, as well as a designer/3D printing specialist. However, we can always use more help on MURPHIE! https://github.com/daveshap/Open_MURPHIE

Please reply here or ping me on LinkedIn!

**Comment by marcothedeveloper123** [comment]
Hey Dave,

I’d love to help you realise ACE with you. Check out my profile at linkedin.com/in/marcobottaro and if you’re interested, let’s chat to see if you feel there’s a fit.

Cheers,

Marco

**Reply by daveshap** [reply]
Hey Marco! This is great. Are you interested in the product owner side of things? I will connect on linkedin
